packages/slate-editor/src/extensions/inline-contacts/transforms/updateInlineContact.ts

```diff
@@ -0,0 +1,14 @@
+import type { ContactNode } from '@prezly/slate-types';
+import type { Editor } from 'slate';
+import { Transforms } from 'slate';
+
+export function updateInlineContact(
+    editor: Editor,
+    contact: ContactNode,
+    patch: Partial<Pick<ContactNode, 'contact'>>,
```

Comment:
Why `Partial<...>`? Is there a point of calling this without providing `contact`?

Reply:
That's an oversight 👍 Initially I wanted to use `patch: Partial<ContactInfo>` but then changed it to `ContactNode` so it's consistent with other transformations. Will fix.